docs/UserGuide.md

```diff
@@ -401,14 +387,41 @@ This can be used in any command where dates are required, such as `incompleteo`
 
 Format: `Day HH:mm`
 
+All potential natural dates that can be used:
+* `Mon HH:mm`
+* `Monday HH:mm`
+* `Tues HH:mm`
+* `Tue HH:mm`
+* `Tuesday HH:mm`
+* `Weds HH:mm`
+* `Wed HH:mm`
+* `Wednesday HH:mm`
+* `Thurs HH:mm`
+* `Thur HH:mm`
+* `Thursday HH:mm`
+* `Fri HH:mm`
+* `Friday HH:mm`
+
 Example of natural date:
 * `Monday 10:30` or `Mon 10:30` will create a datetime in the form of `dd-mm-yyyy HH:mm` that falls on the closest Monday, where `HH:mm` is `10:30`
 * `Sunday 09:30` or `Sun 09:30`will create a datetime in the form of `dd-mm-yyyy HH:mm` that falls on the closest Sunday, where `HH:mm` is `09:30`
-
-Example of natural date in action:
+* If the current time is `7th April 2022 09:30`, inputting `Thurs 08:30` will return `14th April 2022` as the 
+  closest Thursday. 
+  * Reason: `08:30` has already passed the current time, hence ReadyBakey will look for a future date that is a 
+    Thursday instead. 
+* If the current time is `7th April 2022 09:30`, inputting `Thurs 10:30` will return `7th April 2022` as the
+  closest Thursday.
+  * Reason: `10:30` has not passed the current time, hence ReadyBakey will look at the current day as the closest 
+    Thursday.
+
```

Comment:
Can we add one more example for Saturday or Friday, just to show how it works with another date in relation to the given natural date